README.md

```diff
@@ -120,6 +122,7 @@ For syntax, see [lookup.json](./lib/lookup.json), the available attributes are:
 "sha": "<git-commit-sha>"    Test against a specific commit
 "envVar"                     Pass an environment variable before running
 "install": ["--param1", "--param2"] - Array of extra command line parameters passed to 'npm install'
+"tag": ["tag1", "tag2"]      Provide tags so that `citgm-all` can be run on specific tags                
```

Comment:
Maybe something like:

`Provide tags so that `citgm-all` can be run on specific tags` -> `Specify which tags the module comes under`